src/uct/base/uct_md.c

```diff
@@ -24,6 +24,19 @@ ucs_config_field_t uct_md_config_table[] = {
   {NULL}
 };
 
+ucs_config_field_t uct_md_config_rcache_table[] = {
+  {"RCACHE", "try", "Enable using memory registration cache",
```

Comment:
not sure if align should be here too, for now it is unique value for every transport

Reply:
it's possible to set values for nested configs, see https://github.com/openucx/ucx/blob/master/src/uct/ib/dc/base/dc_iface.c#L20

Reply:
yep, I see. what way is correct? set any default value (for now cache line is good candidate)?

Reply:
yes

Reply:
it became more & more complex :)
done